lib/stripe/util.rb

```diff
@@ -147,8 +150,8 @@ def self.symbolize_names(object)
     # involves escaping special characters from parameter keys and values (e.g.
     # `&`).
     def self.encode_parameters(params)
-      Util.flatten_params(params).
-        map { |k,v| "#{url_encode(k)}=#{url_encode(v)}" }.join('&')
+      Util.flatten_params(params)
+          .map { |k, v| "#{url_encode(k)}=#{url_encode(v)}" }.join('&')
```

Comment:
This prefixed dot didn't use to be possible on previous versions of Ruby which has led me to still not really use it, but that said, I kind of like the change.